Fula.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|  
-    s.name              = 'FulaMobile' # Name for your pod
-    s.version           = '0.1.6'
+    s.name              = 'Fula' # Name for your pod
+    s.version           = '1.0.0'
```

Comment:
latest fula version is v1.13.1 right now. but is there a way to read this from the release version instad of hardcoding?

Reply:
or have a aprocess to updat hte versin we publish a new release

Reply:
I mean adding a new flow like this?
```
name: Update Podspec Version

on:
  release:
    types: [published]

jobs:
  update-version:
    runs-on: ubuntu-latest

    steps:
    - name: Check out code
      uses: actions/checkout@v2

    - name: Update version
      run: |
        VERSION=$(echo ${{ github.event.release.tag_name }} | cut -c 2-) # remove the v from the tag name
        sed -i 's/\(s.version\s*=\s*\).*$/\1'"$VERSION"'/' Fula.podspec

    - name: Set up Git
      run: |
        git config user.name "GitHub Actions"
        git config user.email "actions@github.com"

    - name: Commit and push
      run: |
        git add .
        git commit -m "Update version to $VERSION"
        git push
```